README.md

```diff
@@ -63,12 +68,73 @@ Right now just a sketch is imported:
 - sequence steps, generically, with slices for
   - bias
   - dark
-  - gcal (lamp, shutter)
+  - gcal
   - science (offset p/q)
-    - F2 (fpu, pre-imaging, exposure time, filter, lyot wheel, disperser)
+    - F2
 
 There are no other instrument-specific slices for science steps yet.
 
+
+#### Importing Smart Gcal Configuration
+
+Smart Gcal configuration is stored in database tables like everything else instead of in `.csv` files downloaded from SVN as in OCS2.  The `.csv` files can be imported though much like old program `.xml` files.  First, make a symlink to a directory containing the `.csv` files. 
+
+```
+ln -s /path/to/old/smart/gcal/csv smartgcal
+```
+
+If you've ever used the old OT or ODB, you will likely have downloaded the `.csv` files for Smart Gcal. For example, for the production OT you can find them in your home directory:
+
+```
+~/.ocs15/Gemini\ OT\ 2017A.1.1.2_mac/data/jsky.app.ot/smartgcal
+```
+
+Having created the symlink, you can then import from the `sbt` prompt:
+
+```
+sbt> ocs2/runMain gem.ocs2.SmartGcalImporter
+```
+
+#### Import Server
+
+You can run an import server which will import programs or observations on demand directly from a running OCS2 Observing Database.  You start it with
+
+```
+sbt> ocs2/runMain gem.ocs2.ImportServer [odb-hostname]
```

Comment:
Actually, I probably need to update it.  I think there are new F2 filters this semester?